src/content/get-started/using-okteto-cli-and-dashboard.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Using the Okteto CLI and Dashboard
+description: Introduction to the Okteto CLI and Dashboard
+sidebar_label: Using the CLI and Dashboard
+id: using-okteto-cli-and-dashboard
+---
+
+import Image from '@theme/Image';
+
+The Okteto CLI is our [open-source](https://github.com/okteto/okteto) tool that let's you develop your applications on Okteto.
+If you haven't done so yet, install and configure the Okteto CLI following [this guide](get-started/install-okteto-cli.mdx).
+
+Most developers interact with okteto via the Command Line Interface (CLI). Here we’ll explain common CLI commands and a typical workflow for using them.
+
+## Essential Okteto CLI Commands
```

Comment:
We should introduce the Okteto Manifest at this point if we haven't done it before